examples/browser/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CloudproofJS in browser</title>
+    <link rel="icon" href="data:,">
+</head>
+<body>
+    <p>This demo should work without any JS build.</p>
+
+    <script src="node_modules/cloudproof_js/dist/umd/index.js"></script>
+    <script>
+        const { CoverCrypt } = window.cloudproof_js;
+
+        (async () => {
+            const { Policy, PolicyAxis, generateMasterKeys, generateUserSecretKey, encrypt, decrypt } = await CoverCrypt();
+    
+            const policy = new Policy([
+                new PolicyAxis(
+                    "Security Level",
+                    [
+                    { name: "Low", isHybridized: false },
+                    { name: "Medium", isHybridized: false },
+                    { name: "High", isHybridized: true },
+                    ],
+                    true,
+                ),
+                new PolicyAxis(
+                    "Department",
+                    [
+                    { name: "R&D", isHybridized: false },
+                    { name: "HR", isHybridized: false },
+                    ],
+                    false,
+                ),
+            ]);
+
+            const masterKeys = generateMasterKeys(policy);
```

Comment:
we should `console.log` something to show we are doing something :)

Reply:
Why?

Reply:
Done in 3e8c58d8157b0cb5e9e0469d1c14a64ef0d26530